genesyscloud/data_source_genesyscloud_routing_email_domain_test.go

```diff
@@ -0,0 +1,53 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+func TestAccDataSourceRoutingEmailDomain(t *testing.T) {
+	var (
+		emailDomainResourceId   = "email_domain_test"
+		emailDomainId  			= "jcc4234234.thoughtx.com"
+		emailDataResourceId 	= "email_domain_data"
+	)
+
+	fmt.Printf(generateRoutingEmailDomainResource(
```

Comment:
This printf can be removed

Reply:
Doh! Missed that.  Put that in there while debugging

Reply:
Fixed